packages/app/src/app/components/Create/CreateBox/CreateBoxForm.tsx

```diff
@@ -93,66 +143,76 @@ export const CreateBoxForm: React.FC<CreateBoxFormProps> = ({
-                label="Private"
-              />
+              <Select
+                icon={PRIVACY_OPTIONS[permission].icon}
+                defaultValue={permission}
+                onChange={({ target: { value } }) => setPermission(value)}
+              >
+                <option value={0}>{PRIVACY_OPTIONS[0].description}</option>
+                <option value={1}>{PRIVACY_OPTIONS[1].description}</option>
+                <option value={2}>{PRIVACY_OPTIONS[2].description}</option>
+              </Select>
             </Stack>
           </Stack>
 
           <Stack direction="vertical" gap={2}>
             <Text size={3} as="label">
               Open in
             </Text>
-            <Stack direction="vertical" gap={1}>
-              <Radio
-                disabled={disableEditorChange}
-                checked={editor === 'web'}
-                onChange={() => setEditor('web')}
-                label="Web editor"
-              />
-              <Radio
-                disabled={disableEditorChange}
-                checked={editor === 'vscode'}
-                onChange={() => setEditor('vscode')}
-                label="VSCode"
+            {type === 'sandbox' ? (
+              <Input
+                css={{ opacity: 0.7, cursor: 'not-allowed' }}
+                value="CodeSandbox web editor"
+                disabled
               />
-            </Stack>
-            {disableEditorChange && (
+            ) : (
+              <Select
+                icon={EDITOR_ICONS[editor]}
+                defaultValue={editor}
+                onChange={({ target: { value } }) => setEditor(value)}
+              >
+                <option value="csb">CodeSandbox web editor</option>
+                <option value="vscode">
+                  VSCode desktop (Using the CodeSandbox extension)
+                </option>
+              </Select>
+            )}
+
+            {type === 'sandbox' && (
               <Stack gap={1}>
                 <Icon color="#999" name="circleBang" />
                 <Text size={3} variant="muted">
-                  Sandboxes can only be open in the web editor.
+                  Sandboxes can only be open in the web editor.{' '}
+                  <Text
+                    as="a"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    href={docsUrl('/learn/sandboxes/editor-features')}
+                  >
+                    Learn more
+                  </Text>{' '}
+                  about sandboxes.
                 </Text>
               </Stack>
             )}
           </Stack>
 
-          {showVMSpecs && (
-            <Stack direction="vertical" align="flex-start" gap={2}>
-              <Text size={3} as="label">
-                Virtual machine specifications
+          <Stack direction="vertical" align="flex-start" gap={2}>
+            <Text size={3} as="label">
+              Runtime
```

Comment:
Could you tell me what the reasoning is behind this change?
To me, "Virtual machine specifications" or "Virtual machine type" looks like a better description than "Runtime", which feels more abstract. It's not a dealbreaker, but I think it would be a little bit better.

Reply:
I added this field for sandboxes and the value is `Browser`, so Ives suggested `Runtime` would be more appropriate

Reply:
ah, because it's not a virtual machine?
I guess that makes sense, it is more apt. Let's leave it like this.